xcube_sh/store.py

```diff
@@ -306,30 +306,33 @@ def get_data_store_params_schema(cls) -> JsonObjectSchema:
         )
 
     @classmethod
-    def get_type_ids(cls) -> Tuple[str, ...]:
-        return TYPE_ID_DATASET,
+    def get_type_specifiers(cls) -> Tuple[str, ...]:
+        return str(TYPE_SPECIFIER_CUBE),
 
-    def get_data_ids(self, type_id: str = None) -> Iterator[Tuple[str, Optional[str]]]:
-        self._assert_valid_type_id(type_id)
-        metadata = SentinelHubMetadata()
-        for data_id, dataset in metadata.datasets.items():
-            yield data_id, dataset.get('title')
+    def get_type_specifiers_for_data(self, data_id: str) -> Tuple[str, ...]:
+        return self.get_type_specifiers()
 
-    def has_data(self, data_id: str) -> bool:
+    def get_data_ids(self, type_specifier: str = None, include_titles=True) -> Iterator[Tuple[str, Optional[str]]]:
+        if self._is_supported_type_specifier(type_specifier):
+            metadata = SentinelHubMetadata()
+            for data_id, dataset in metadata.datasets.items():
+                yield data_id, dataset.get('title')
```

Comment:
```suggestion
                yield data_id, (dataset.get('title') if include_titles else None)
```

The docstring for the `get_data_ids` abstract method says "If *include_titles* is false, the second item of the result tuple will be None.".